documentation/porting_guidelines.md

```diff
@@ -216,7 +216,7 @@ device is publicly available or not.
 
 #### 2.2.2 Update travis.xml
 Since we are using Travis to test pull request etc, we would like that you also
-all your device to the [travis] file, so that it will at least be built when
+add your device to the [travis] file, so that it will at least be built when
```

Comment:
Actually this is outdated. Travis builds and runs the tests for the QEMU platform. Builds for other platforms are now handled by Shippable (`.shippable.yml`).